views/js/controller/DeliveryMgmt/editDelivery.js

```diff
@@ -20,13 +20,56 @@
 define([
     'jquery',
     'util/url',
+    'services/features',
     'ui/modal',
     'css!taoDeliveryRdfCss/delivery-rdf.css'
-], function ($, urlUtil) {
+], function ($, urlUtil, features) {
     'use strict';
 
+    /**
+     * Finds property block and hides it adding .hidden class
+     * @param {string} propName 
+     */
+    function hidePropertyBlockByName(propName) {
+        const inputCssQuery = `form[action="/taoDeliveryRdf/DeliveryMgmt/editDelivery"] input[name="${propName}"]`;
+        $(inputCssQuery).closest('form > div').addClass('hidden');
+    }
+
     return {
         start(){
+
+            const featuresPath = 'taoDeliveryRdf/deliveryMgmt/';
+            if(!features.isVisible(`${featuresPath}resourceIdentifier`)) {
+                hidePropertyBlockByName('id');
+            }
+            if(!features.isVisible(`${featuresPath}maxExecutions`)) {
```

Comment:
@shaveko this form is built in the BE side, I am not sure why we are hiding the fields in the FE. 

We should probably have done this in the BE https://github.com/oat-sa/extension-tao-delivery-rdf/blob/develop/model/Delivery/Presentation/Web/Form/DeliveryFormFactory.php#L53

I am also concerned on the side effects of removing it in the FE side, cause we are still considering those fields in the BE and I am not sure the integrity of the data will be consistent after submitting this form.